src/main/java/com/epam/academy/uglyCode/service/LanguageServiceImpl.java

```diff
@@ -1,49 +1,47 @@
 package com.epam.academy.uglyCode.service;
 
-import static com.epam.academy.common.Language.HTML;
-import static com.epam.academy.common.Language.JS;
-import static com.epam.academy.common.Language.PYTHON;
+import com.epam.academy.common.Language;
+import com.epam.academy.common.LanguageService;
 
 import java.security.InvalidParameterException;
 
-import com.epam.academy.common.Language;
-import com.epam.academy.common.LanguageService;
+import static com.epam.academy.common.Language.*;
 
-public final class LanguageServiceImpl implements LanguageService{
+public final class LanguageServiceImpl implements LanguageService {
 
-	@Override
-	public boolean isProgrammingLangugage(Language language) {
-		if (language == HTML) {
-			return false;
-		} else {
-			return true;
-		}
-	}
+    @Override
+    public boolean isProgrammingLanguage(Language language) {
+        return language != HTML;
+    }
 
-	@Override
-	public boolean canUseForScripting(Language language) {
-		if (language == JS || language == PYTHON) {
-			return true;
-		} else {
-			return false;
-		}
-	}
+    @Override
+    public boolean canUseForScripting(Language language) {
+        return language == JS || language == PYTHON;
+    }
 
-	@Override
-	public String getQualifiedName(Language language) {
-		switch(language) {
-		case JS:
-			return "Java Script";
-		case JAVA:
-			return "Java (Oracle)";
-		case KOTLIN:
-			return "Kotlin ('NetBrains')";
-		case HTML:
-			return "Hyper Text Markup Language";
-		case PYTHON:
-			return "Python (the all in all)";
-		}
-		
-		throw new InvalidParameterException(String.format("Language %s is not defined", language.name()));
-	}
+    @Override
+    public String getQualifiedName(Language language) throws InvalidParameterException {
+        String qualifiedName;
+        switch (language) {
+            case JS:
+                qualifiedName = "Java Script";
+                break;
+            case JAVA:
+                qualifiedName = "Java (Oracle)";
+                break;
+            case KOTLIN:
+                qualifiedName = "Kotlin ('NetBrains')";
+                break;
+            case HTML:
+                qualifiedName = "Hyper Text Markup Language";
+                break;
+            case PYTHON:
+                qualifiedName = "Python (the all in all)";
+                break;
+            default:
+                throw new InvalidParameterException(String.format("Language %s is not defined",
+                        language.name()));
+        }
+        return qualifiedName;
+    }
 }
```

Comment:
Still the nested control loop or switch-case statements are going to be tough to mainitain. What if a new language comes in to this picture ? One way to simplify is to make use of the `enum` class and add attributes to them, which can be re-used throughout and easy to maintain